recipes-go/3mdeb-rtectrl/3mdeb-rtectrl_git.bbappend

```diff
@@ -17,6 +27,15 @@ do_install:append() {
 
     install -d ${D}${systemd_unitdir}/system/
     install -m 0644 ${WORKDIR}/RteCtrl.service ${D}${systemd_unitdir}/system/
+
+    # We need flash.sh only on images with distro versions 0.8.x:
```

Comment:
Not needed. Maybe we misunderstood. It will be included in v0.8.x and later version as well. As long as we work through this second ticket: https://github.com/3mdeb/meta-rte/issues/82

Then, we will simply drop the flash.sh as not necessary anymore.

Reply:
Dropped the commit [here](https://github.com/3mdeb/meta-rte/compare/c4c0f2b6c947061defa1b83bd585745ec0ecadb6..b09e9b726f3372dd7e3b9054bc79468968f7ece0)